.eslintignore

```diff
@@ -0,0 +1,2 @@
+lib/cover-service-api/model/
+lib/cover-service-api/cover-service.ts
```

Comment:
I think, because we use prettier to format code - the formatting is still done on "save". 
As far as I know, ESlint doesn't really format code in any case. It only points out when it's not living up to the standards.